Fix documentation build errors with Perl 5.18 pod2man

This fixes errors building man pages with newer versions of pod2man
included with Perl 5.18.

Upstream-Status: Submitted
Signed-off-by: Jonathan Liu

Backported from upstream commit 8792b7f to openssl-0.9.8y
by Andreas Frisch <fraxinas@opendreambox.org>.

.pod~ ./doc/apps/c_rehash.pod
Index: openssl-0.9.8za/doc/apps/c_rehash.pod
===================================================================
--- openssl-0.9.8za.orig/doc/apps/c_rehash.pod
+++ openssl-0.9.8za/doc/apps/c_rehash.pod
@@ -42,12 +42,12 @@ The name (and path) of an executable to
 
 Colon separated list of directories to operate on. Ignored if directories are listed on the command line.
 
+=back
+
 =head1 SEE ALSO
 
 L<openssl(1)|openssl(1)>, L<x509(1)|x509(1)>
 
-=back
-
 =head1 BUGS
 
 No known bugs
Index: openssl-0.9.8za/doc/apps/smime.pod
===================================================================
--- openssl-0.9.8za.orig/doc/apps/smime.pod
+++ openssl-0.9.8za/doc/apps/smime.pod
@@ -265,28 +265,28 @@ encrypted data is used for other purpose
 
 =over 4
 
-=item 0
+=item Z<>0
 
 the operation was completely successfully.
 
-=item 1 
+=item Z<>1 
 
 an error occurred parsing the command options.
 
-=item 2
+=item Z<>2
 
 one of the input files could not be read.
 
-=item 3
+=item Z<>3
 
 an error occurred creating the PKCS#7 file or when reading the MIME
 message.
 
-=item 4
+=item Z<>4
 
 an error occurred decrypting or verifying the message.
 
-=item 5
+=item Z<>5
 
 the message was verified correctly but an error occurred writing out
 the signers certificates.
Index: openssl-0.9.8za/doc/ssl/SSL_COMP_add_compression_method.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_COMP_add_compression_method.pod
+++ openssl-0.9.8za/doc/ssl/SSL_COMP_add_compression_method.pod
@@ -53,11 +53,11 @@ SSL_COMP_add_compression_method() may re
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The operation succeeded.
 
-=item 1
+=item Z<>1
 
 The operation failed. Check the error queue to find out the reason.
 
Index: openssl-0.9.8za/doc/ssl/SSL_CTX_add_session.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_CTX_add_session.pod
+++ openssl-0.9.8za/doc/ssl/SSL_CTX_add_session.pod
@@ -52,13 +52,13 @@ The following values are returned by all
 
 =over 4
 
-=item 0
+=item Z<>0
 
  The operation failed. In case of the add operation, it was tried to add
  the same (identical) session twice. In case of the remove operation, the
  session was not found in the cache.
 
-=item 1
+=item Z<>1
  
  The operation succeeded.
 
Index: openssl-0.9.8za/doc/ssl/SSL_CTX_load_verify_locations.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_CTX_load_verify_locations.pod
+++ openssl-0.9.8za/doc/ssl/SSL_CTX_load_verify_locations.pod
@@ -100,13 +100,13 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The operation failed because B<CAfile> and B<CApath> are NULL or the
 processing at one of the locations specified failed. Check the error
 stack to find out the reason.
 
-=item 1
+=item Z<>1
 
 The operation succeeded.
 
Index: openssl-0.9.8za/doc/ssl/SSL_CTX_set_client_CA_list.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_CTX_set_client_CA_list.pod
+++ openssl-0.9.8za/doc/ssl/SSL_CTX_set_client_CA_list.pod
@@ -66,13 +66,13 @@ values:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 A failure while manipulating the STACK_OF(X509_NAME) object occurred or
 the X509_NAME could not be extracted from B<cacert>. Check the error stack
 to find out the reason.
 
-=item 1
+=item Z<>1
 
 The operation succeeded.
 
Index: openssl-0.9.8za/doc/ssl/SSL_CTX_set_session_id_context.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_CTX_set_session_id_context.pod
+++ openssl-0.9.8za/doc/ssl/SSL_CTX_set_session_id_context.pod
@@ -64,13 +64,13 @@ return the following values:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The length B<sid_ctx_len> of the session id context B<sid_ctx> exceeded
 the maximum allowed length of B<SSL_MAX_SSL_SESSION_ID_LENGTH>. The error
 is logged to the error stack.
 
-=item 1
+=item Z<>1
 
 The operation succeeded.
 
Index: openssl-0.9.8za/doc/ssl/SSL_CTX_set_ssl_version.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_CTX_set_ssl_version.pod
+++ openssl-0.9.8za/doc/ssl/SSL_CTX_set_ssl_version.pod
@@ -42,11 +42,11 @@ and SSL_set_ssl_method():
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The new choice failed, check the error stack to find out the reason.
 
-=item 1
+=item Z<>1
 
 The operation succeeded.
 
Index: openssl-0.9.8za/doc/ssl/SSL_accept.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_accept.pod
+++ openssl-0.9.8za/doc/ssl/SSL_accept.pod
@@ -44,13 +44,13 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
 return value B<ret> to find out the reason.
 
-=item 1
+=item Z<>1
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
Index: openssl-0.9.8za/doc/ssl/SSL_clear.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_clear.pod
+++ openssl-0.9.8za/doc/ssl/SSL_clear.pod
@@ -56,12 +56,12 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The SSL_clear() operation could not be performed. Check the error stack to
 find out the reason.
 
-=item 1
+=item Z<>1
 
 The SSL_clear() operation was successful.
 
Index: openssl-0.9.8za/doc/ssl/SSL_connect.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_connect.pod
+++ openssl-0.9.8za/doc/ssl/SSL_connect.pod
@@ -41,13 +41,13 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
 return value B<ret> to find out the reason.
 
-=item 1
+=item Z<>1
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
Index: openssl-0.9.8za/doc/ssl/SSL_do_handshake.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_do_handshake.pod
+++ openssl-0.9.8za/doc/ssl/SSL_do_handshake.pod
@@ -45,13 +45,13 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
 return value B<ret> to find out the reason.
 
-=item 1
+=item Z<>1
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
Index: openssl-0.9.8za/doc/ssl/SSL_read.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_read.pod
+++ openssl-0.9.8za/doc/ssl/SSL_read.pod
@@ -86,7 +86,7 @@ The following return values can occur:
 The read operation was successful; the return value is the number of
 bytes actually read from the TLS/SSL connection.
 
-=item 0
+=item Z<>0
 
 The read operation was not successful. The reason may either be a clean
 shutdown due to a "close notify" alert sent by the peer (in which case
Index: openssl-0.9.8za/doc/ssl/SSL_session_reused.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_session_reused.pod
+++ openssl-0.9.8za/doc/ssl/SSL_session_reused.pod
@@ -27,11 +27,11 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 A new session was negotiated.
 
-=item 1
+=item Z<>1
 
 A session was reused.
 
Index: openssl-0.9.8za/doc/ssl/SSL_set_fd.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_set_fd.pod
+++ openssl-0.9.8za/doc/ssl/SSL_set_fd.pod
@@ -35,11 +35,11 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The operation failed. Check the error stack to find out why.
 
-=item 1
+=item Z<>1
 
 The operation succeeded.
 
Index: openssl-0.9.8za/doc/ssl/SSL_set_session.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_set_session.pod
+++ openssl-0.9.8za/doc/ssl/SSL_set_session.pod
@@ -37,11 +37,11 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The operation failed; check the error stack to find out the reason.
 
-=item 1
+=item Z<>1
 
 The operation succeeded.
 
Index: openssl-0.9.8za/doc/ssl/SSL_set_shutdown.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_set_shutdown.pod
+++ openssl-0.9.8za/doc/ssl/SSL_set_shutdown.pod
@@ -24,7 +24,7 @@ The shutdown state of an ssl connection
 
 =over 4
 
-=item 0
+=item Z<>0
 
 No shutdown setting, yet.
 
Index: openssl-0.9.8za/doc/ssl/SSL_shutdown.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_shutdown.pod
+++ openssl-0.9.8za/doc/ssl/SSL_shutdown.pod
@@ -92,19 +92,19 @@ The following return values can occur:
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The shutdown is not yet finished. Call SSL_shutdown() for a second time,
 if a bidirectional shutdown shall be performed.
 The output of L<SSL_get_error(3)|SSL_get_error(3)> may be misleading, as an
 erroneous SSL_ERROR_SYSCALL may be flagged even though no error occurred.
 
-=item 1
+=item Z<>1
 
 The shutdown was successfully completed. The "close notify" alert was sent
 and the peer's "close notify" alert was received.
 
-=item -1
+=item Z<>-1
 
 The shutdown was not successful because a fatal error occurred either
 at the protocol level or a connection failure occurred. It can also occur if
Index: openssl-0.9.8za/doc/ssl/SSL_write.pod
===================================================================
--- openssl-0.9.8za.orig/doc/ssl/SSL_write.pod
+++ openssl-0.9.8za/doc/ssl/SSL_write.pod
@@ -79,7 +79,7 @@ The following return values can occur:
 The write operation was successful, the return value is the number of
 bytes actually written to the TLS/SSL connection.
 
-=item 0
+=item Z<>0
 
 The write operation was not successful. Probably the underlying connection
 was closed. Call SSL_get_error() with the return value B<ret> to find out,
